src/index.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import playerLoader from '@brightcove/player-loader';
+
+/**
+ * The official React component for the Brightcove
+ * Player! This uses `@brightcove/player-loader` to load
+ * a player into a React component based on the given props.
+ */
+class ReactBrightcovePlayer extends React.Component {
+
+  /**
+   * Create a new Brightcove player.
+   *
+   * @param {Object} props
+   *        Most options will be passed along to player-loader, except for
+   *        options that are listed, and `refNode`/`refNodeInsert`
+   *        which will be ignored, as they are used internally to
+   *        get this component to work.
+   */
+  constructor(props) {
+    super(props);
+    this.state = {};
+    this.refNode = React.createRef();
```

Comment:
the ref node is how to get react to basically ignore things that are happening inside, right?